guides/upstream_filename_to_link.json

```diff
@@ -0,0 +1,18 @@
+{
+  "build/Administering_Project/index-foreman-el.html": "administering_project",
+  "build/Configuring_Load_Balancer/index-foreman-el.html": "configuring_capsules_with_a_load_balancer",
+  "build/Deploying_Project_on_AWS/index-foreman-el.html": "deploying_project_on_amazon_web_services",
+  "build/Installing_Proxy/index-foreman-el.html": "installing_capsule_server",
+  "build/Installing_Server/index-foreman-el.html": "installing_satellite_server_in_a_connected_network_environment",
+  "build/Installing_Server_Disconnected/index-foreman-el.html": "installing_satellite_server_in_a_disconnected_network_environment",
+  "build/Managing_Configurations_Ansible/index-foreman-el.html": "managing_configurations_using_ansible_integration_in_project",
+  "build/Managing_Configurations_Puppet/index-foreman-el.html": "managing_configurations_using_puppet_integration_in_project",
+  "build/Managing_Content/index-foreman-el.html": "managing_content",
+  "build/Managing_Hosts/index-foreman-el.html": "managing_hosts",
+  "build/Provisioning_Hosts/index-foreman-el.html": "provisioning_hosts",
+  "build/Planning_for_Project/index-foreman-el.html": "satellite_overview_concepts_and_deployment_considerations",
+  "build/Tuning_Performance/index-foreman-el.html": "tuning_performance_of_project",
+  "build/Managing_Security_Compliance/index-foreman-el.html": "managing_security_compliance",
+  "build/Upgrading_Project/index-foreman-el.html": "upgrading_project",
```

Comment:
Upgrading has been split further to connected and disconnected. https://github.com/theforeman/foreman-documentation/pull/2599

Reply:
Done